app/controllers/api_controller.rb

```diff
@@ -21,6 +21,14 @@ def uid
     end
   end
 
+  def userByUid
+    uid  = params[:uid]
+    ring = Ring.find_by! uid: uid
+    user = ring.user
+    response = { status: :ok, user: user }
```

Comment:
This response object is what you return in the API. So if you want this to just be the twitter username, do that. Also, "status OK" will be conveyed by the HTTP 200 response code, so no need for that.

Instead of returning whole user object, go with `user.twitter.try(:username)`, which will return nil if the user doesn't have an associated twitter record.